adjaceny-list-graph/adjacency-list-graph.go

```diff
@@ -0,0 +1,327 @@
+// Copyright 2018 Anthony A. Abeo. All rights reserved.
+// Use of this source code is governed by a `MIT License`.
+// license that can be found in the LICENSE file.
+
```

Comment:
Take out the copy right section from the code

Reply:
Sure, I will take delete the copyright section.

I have submitted a fix for the bubblesort error in a different PR.
Should I uncomment it in the graph?